packages/rolldown/tests/fixtures/builtin-plugin/glob-import/basic/_config.ts

```diff
@@ -1,11 +1,11 @@
-import { BuiltinGlobImportPlugin } from 'rolldown'
+import { builtinGlobImportPlugin } from 'rolldown'
 import { defineTest } from '@tests'
 import * as path from 'path'
 
 export default defineTest({
   config: {
     plugins: [
-      new BuiltinGlobImportPlugin({
+      builtinGlobImportPlugin({
```

Comment:
I don't think we need to emphasize the `builtin` prefix here. These plugins are __imported__ from `rolldown`, which already has the builtin semantic.

Could you remove it? Other changes are fine.